org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -25,13 +25,13 @@
             <unit id="org.mockito.mockito-core" version="0.0.0"/>
             <unit id="org.apache.commons.commons-io" version="0.0.0"/>
             <unit id="org.eclipse.jdt.apt.pluggable.core" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/eclipse/updates/4.31/"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.32-I-builds/I20240324-1800/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.jdt.core.compiler.batch" version="0.0.0"/>
             <unit id="org.eclipse.jdt.core" version="0.0.0"/>
             <unit id="org.eclipse.jdt.apt.core" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/jdtls/jdt-core-incubator/snapshots/"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.32-I-builds/I20240324-1800/"/>
```

Comment:
Once we verify this change works well, we should undo this particular line, and I will manually rebase the `develop` branch at https://github.com/eclipse-jdtls/eclipse-jdt-core-incubator so that it simply tracks the same commit as `I20240324-1800`.

Reply:
Updating to I20240325-1800 required more changes, I added a commit for that

Reply:
I introduced those changes :stuck_out_tongue_winking_eye: . Setting them to `true` instead of `false` would fix https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2371#issuecomment-2010191794 . We can do it in a separate PR to avoid confusion though.

Reply:
Fixing #2371 requires setting the boolean value from the preferences. Putting `true` by default would probably piss off a bunch of folks.

Reply:
What about if we did it [just for fields](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2371#issuecomment-1351763478) ? Not a huge fan of another setting for the reason that people won't really spend the time to discover it.

Reply:
unfortunately, I think putting final by default will likely be pretty polarizing. A setting is the least bad option IMO, until we can provide a way to let users set it via some wizard.